### PR TITLE
Adds filter reset buttons for agent and complexity

### PR DESCRIPTION
**Pull request by elvisyjlin**
Please test it. Thank you!

### CODE REVIEW

**Comment by pschydlo** [comment]
wow! Great code! 

There were two parts I was not sure about what the best solution would be:

1. How to handle the divider 
2. How to handle the resetting of the drop-down text

You handled them both perfectly! 

### Regarding the first point, the divider: 

The solution to add the divider to the options list makes the usage of the divider very flexible and will be great for any new dropdowns that will be added down the road!

I have just a minor concern with the name, calling it `<hr/>` might create confusion in the future about the usage of this option, it might create the impression that one could use arbitrary HTML here. A solution would be to give it a more "high-level" name like `:divider:`, what do you think would be a good name?

### Regarding the second point, the resetting of the drop-down name:

The solution is amazingly executed! I had thought about just leaving it at "all", but resetting the field to the original name is more elegant, great solution!

### TL;DR
Everything looks clean and tidy! I would just recommend we change the name of the divider option from `<hr/>` to a name like `:divider:`

Thanks again for the great addition!

All the best,
Paul

**Comment by elvisyjlin** [comment]
Oh yeah I gave the name of the divider `<hr/>` because it looks like that HTML element in the rendered page. `:divider:` sounds much better and I think this is a more suitable name for the divider.

**Comment by pschydlo** [comment]
Perfect! Merged! 

If you notice any other feature that would make the environment's exploration more intuitive you are more than welcome to submit another issue!

Wish you a great week! 

All the best,
Paul
